modular_skyrat/modules/modular_ert/code/marine_override/modified_equipment.dm

```diff
@@ -1,9 +1,54 @@
-/datum/outfit/centcom/ert/marine
-	suit_store = /obj/item/gun/ballistic/automatic/cmg
-	belt = /obj/item/storage/belt/military/assault/full/cmg
-
-/datum/outfit/centcom/ert/marine/medic
-	suit_store = /obj/item/storage/belt/holster/detective/full/ert/skyrat
-
-/datum/outfit/centcom/ert/marine/engineer
-	suit_store = /obj/item/gun/ballistic/shotgun/automatic/as2
+/datum/outfit/centcom/ert/marine //commander
+	suit_store = /obj/item/gun/ballistic/automatic/ar/modular/m44a/grenadelauncher
+	belt = /obj/item/storage/belt/military/assault/full/m44a
+	back = /obj/item/mod/control/pre_equipped/marine
+	backpack_contents = list(
+		/obj/item/storage/box/survival/engineer/marine = 1,
+		/obj/item/melee/baton/security/loaded = 1,
+		/obj/item/storage/box/handcuffs = 1,
+		/obj/item/ammo_box/a40mm = 2, //gods must be strong
+	)
+	l_hand = null
+	r_hand = null
+	
+/datum/outfit/centcom/ert/marine/security //generic and/or heavy
+	suit_store = /obj/item/gun/ballistic/automatic/ar/modular/m44a/shotgun
+	belt = /obj/item/storage/belt/military/assault/full/m44a
+	back = /obj/item/mod/control/pre_equipped/marine
+	backpack_contents = list(
+		/obj/item/storage/box/survival/engineer/marine = 1,
+		/obj/item/melee/baton/security/loaded = 1,
+		/obj/item/storage/box/handcuffs = 1,
+		/obj/item/ammo_box/advanced/s12gauge/buckshot/marine = 2,
+	)
+	l_hand = null
+	r_hand = null
+	
+/datum/outfit/centcom/ert/marine/medic //medic
+	suit_store = /obj/item/gun/ballistic/automatic/ar/modular/m44a/scoped
+	belt = /obj/item/storage/belt/military/assault/full/m44a
+	back = /obj/item/mod/control/pre_equipped/marine
+	backpack_contents = list(
+		/obj/item/storage/box/survival/engineer/marine = 1,
+		/obj/item/reagent_containers/hypospray/combat = 1,
+		/obj/item/storage/medkit/regular = 1,
```

Comment:
why set `= 1` on only some of these?

Reply:
It’s what their kit normally included. I can totally throw in more, though

Reply:
Ah wait yes I am retarded